lib/fortnox/api/models/base.rb

```diff
@@ -30,6 +30,17 @@ def self.new( hash )
           IceNine.deep_freeze( obj )
         end
 
+        # This filtering logic could be improved since it is currently O(N*M).
+        def attributes( *options )
```

Comment:
We should add tests for this this method.

Reply:
Yepp.